docs/custom_config.md

```diff
@@ -0,0 +1,132 @@
+# Custom Configuration
```

Comment:
This is super nitpicky, but I see that `custom_config.md` uses an underscore, and `runner-types.md` uses a dash. I don't have a strong preference on whether snake_case or kebab-case is used, but could we use one style for all the doc names?

Reply:
Oops, thank you for catching that!